docs/api/ctx.md

```diff
@@ -1000,6 +1000,34 @@ app.Get("/admin", func(c fiber.Ctx) error {
 })
 ```
 
+An alternative version of the Locals method that takes advantage of Go's generics feature is also available. This version 
+allows for the manipulation and retrieval of local values within a request's context with a more specific data type.
+
+```go title="Signature"
+func Locals[V any](c Ctx, key any, value ...any) V
+```
+
+```go title="Example"
+app.Use(func(c Ctx) error {
+    fiber.Locals[string](c, "john", "doe")
+    fiber.Locals[int](c, "age", 18)
+    fiber.Locals[bool](c, "isHuman", true)
+    return c.Next()
+})
+app.Get("/test", func(c Ctx) error {
+	fiber.Locals[string](c, "john")     // "doe"
+    fiber.Locals[int](c, "age")         // 18
+    fiber.Locals[bool](c, "isHuman")    // true
+return nil
+})
+````
+
+Keep in mind, you will need Go 1.18 or later to take advantage of generics. Also, be aware of the types you pass into 
```

Comment:
No need to write `Keep in mind, you will need Go 1.18 or later to take advantage of generics. ` v3 will support only latest 2 versions of Go

Reply:
Thx, i'll remove it